Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -94,6 +94,9 @@ struct FPTIBatchData: Codable {
             #endif
         }()
 
+        /// The type of link the SDK will be handling, currently deeplink or universal
+        let linkType: String?
```

Comment:
Take it or leave it - using an enum here to enforce we only want these 2 possible string values to be sent for this param. Also would move the string logic for what this field could be out of VenmoClient